### ISSUE TITLE
Server Crash: NoClassDefFoundError: nex/init/NetherExParticleTypes

### ISSUE THREAD

**Issue by gazpachoking** [LogicTechCorp/NetherEx-Archive#117]
Multiplayer server crashed, here is the error. Version 1.12.2-2.0.7 Let me know if you need any more info.
```
Description: Ticking entity

java.lang.NoClassDefFoundError: nex/init/NetherExParticleTypes
        at nex.world.ExplosionSpore.func_77279_a(ExplosionSpore.java:215)
        at nex.entity.monster.EntitySporeCreeper.explode(EntitySporeCreeper.java:261)
        at nex.entity.monster.EntitySporeCreeper.func_70071_h_(EntitySporeCreeper.java:187)
        at net.minecraft.world.World.func_72866_a(World.java:1990)
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:832)
        at net.minecraft.world.World.func_72870_g(World.java:1952)
        at net.minecraft.world.World.func_72939_s(World.java:1756)
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:613)
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:767)
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:397)
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:668)
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.ClassNotFoundException: nex.init.NetherExParticleTypes
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:191)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
        ... 13 more
Caused by: net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerException: Exception in class transformer net.minecraftforge.fml.common.asm.transformers.SideTransformer@7ac9af2a from coremod
FMLCorePlugin
        at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:260)
        at net.minecraft.launchwrapper.LaunchClassLoader.runTransformers(LaunchClassLoader.java:279)
        at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:176)
        ... 15 more
Caused by: java.lang.RuntimeException: Attempted to load class nex/init/NetherExParticleTypes for invalid side SERVER
        at net.minecraftforge.fml.common.asm.transformers.SideTransformer.transform(SideTransformer.java:62)
        at net.minecraftforge.fml.common.asm.ASMTransformerWrapper$TransformerWrapper.transform(ASMTransformerWrapper.java:256)
        ... 17 more
```

**Comment by LogicTechCorp** [comment]
This issue has been fixed in LEX 1.0.8 and NEX 1.0.8. Just waiting on Curse to approve them.

**Comment by gazpachoking** [comment]
Awesome, thanks!
